backend/test/integration/test_schedule.py

```diff
@@ -893,6 +898,17 @@ def test_confirm(
             assert slot.booking_status == models.BookingStatus.booked
             assert appointment.status == models.AppointmentStatus.closed
 
+        # Now try to confirm the same event again
+        response = with_client.put(
+            '/schedule/public/availability/booking',
+            json=availability,
+            headers=auth_headers,
+        )
+
+        with with_db() as db:
+            assert response.status_code == 404, response.text
```

Comment:
I don't think this requires the with_db() as db line.

Reply:
That is undoubtedly true.